handlers/PivotalHandler.ts

```diff
@@ -115,6 +134,75 @@ class PivotalHandler {
     });
   }
 
+  // Create a single review
+  static async createReview({ apiKey, projectId, review }: ReviewActionParams): Promise<void> {
+    // We should delete id and reviewer_id to meet API requirements
+    delete review.id;
```

Comment:
I think you can do this using es6 to return a new object without the id property and it won't be a mutation like this.

Reply:
Yep, we should keep this immutable